docs/reference/sql/copy.md

```diff
@@ -15,5 +17,54 @@ data (`/xxx/xxx/output.parquet` in this case).
 
 `WITH` adds options such as the file `FORMAT` which specifies the format of the exported file. In this example, the format is Parquet; it is a columnar storage format used for big data processing. Parquet efficiently compresses and encodes columnar data for big data analytics. 
 
-# COPY FROM
-// TODO
+## COPY FROM
+
+`COPY FROM` is used to import data from a file into a table.
+
+The syntax for using `COPY FROM` is as follows:
+
+```sql
+COPY [<db>.]<table_name>
+FROM { '<path>/[<filename>]' }
+[ [ WITH ]
+ (
+   [ FORMAT =  { parquet } ]
+   [ PATTERN = '<regex_pattern>' ]
+ )
+]
+[ [ CONNECTION ]
+ (
+   [ ENDPOINT = '<uri>' ]
+   [ ACCESS_KEY_ID = '<key_id>' ]
+   [ SECRET_ACCESS_KEY = '<access_key>' ]
+   [ SESSION_TOKEN = '<token>' ]
+   [ REGION = '<region>' ]
+   [ ENABLE_VIRTUAL_HOST_STYLE = '<boolean>']
+   ..
+ )
+]
+```
+
+The command starts with the keyword `COPY`, followed by the name of the table you want to import data into.
+
+`FORMAT` specifies the file format of the imported file. In this example, the format is Parquet.
+
+`PATTERN` option can be used with wildcard characters like * to specify multiple input files that 
+match a certain pattern. For example, you can use the following syntax to import all files in the 
+directory "/path/to/folder" with the name who contains `parquet`:
+
+```sql
+COPY tbl FROM '/path/to/folder/' WITH (PATTERN = '.*parquet.*');
+```
+
+`CONNECTION` options are used when importing data from cloud storage services such as AWS S3. These options include:
```

Comment:
Copy from S3 is not tested yet. We might need to test it manually.

Reply:
Testing dependency on the [CONNECTION clause of Copy To](https://github.com/GreptimeTeam/greptimedb/pull/1163), we can merge this after it is completed.

Reply:
Let's test this feature and make sure it works, and then merge this pull request. We could review and merge  https://github.com/GreptimeTeam/greptimedb/pull/1163 first.